composer/trainer/checkpoint.py

```diff
@@ -18,6 +18,26 @@
 log = logging.getLogger(__name__)
 
 
+def parse_checkpoint_filepath(checkpoint_filepath: str):
```

Comment:
Discussed offline to tarball the checkpoint. This will help ensure 1) a checkpoint doesn't get corrupted in transit (e.g. you somehow download some of the files but not all) and 2) object stores use a flat directory structure, and it can get expensive to list all files that begin with a prefix.